src/pages/components/toggletip/usage.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: Toggletip
+description:
+  Toggletips use the disclosure pattern to toggle the visibility of a popover.
+  They display additional, hidden information upon the click of a UI trigger
+  element and can contain interactive elements.
+tabs: ['Usage', 'Style']
+---
+
+<PageDescription>
+
+Toggletips use the disclosure pattern to toggle the visibility of a popover.
```

Comment:
I think you should actually inverse the sentence order here. This first sentence uses a lot of terms I might not know yet as a user and doesn't really help me understand what a toogletip is.